supabase/migrations/20260815010000_resonance_reconcile.sql

```
@@ -0,0 +1,46 @@
+update public.workflows
+set slug = lower(regexp_replace(regexp_replace(name, '[^a-zA-Z0-9]+', '-', 'g'), '(^-|-$)', '', 'g'))
+where slug is null or slug = '';
+
+update public.workflow_runs r
+set project_id = w.project_id
+from public.workflows w
+where r.project_id is null and r.workflow_id = w.id;
+
+alter table public.workflows alter column slug set not null;
+create unique index if not exists idx_workflows_project_slug on public.workflows(project_id, slug);
+create index if not exists idx_workflow_runs_project_created on public.workflow_runs(project_id, created_at desc);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)